### PR TITLE
[Hub Generated] Review request for Microsoft.ResourceHealth to add version preview/2023-07-01-preview

### PR DESCRIPTION
**Pull request by mkatkam**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://openapihub.azure-devex-tools.com/branch/Azure/azure-rest-api-specs/mkatkam-resourcehealth-Microsoft.ResourceHealth-2023-07-01-preview...main/).
# ARM (Control Plane) API Specification Update Pull Request 

## PR review workflow diagram

Please understand this diagram before proceeding. It explains how to get your PR approved & merged.

diagram

## Purpose of this PR

What's the purpose of this PR? Check all that apply. This is **mandatory**!

  - [x] New API version. (Such PR should have been generated with [OpenAPI Hub](https://aka.ms/openapihub), per [this wiki doc](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).)
  - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
  - [ ] Update existing version to fix swagger quality issues in S360.
  - [ ] Other, please clarify:
    - _edit this with your clarification_

## Due diligence checklist

To merge this PR, you **must** go through the following checklist and confirm you understood 
and followed the instructions by checking all the boxes:

- [x] I have reviewed the general guidance on the spec PR review process: https://aka.ms/specprreview.
- [x] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data-plane related specifications.
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy).
- [x] I have reviewed following [Resource Provider guidelines](https://aka.ms/rpguidelines), including
  [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can request review from an ARM API Review board.

### ARM API changes review

- If you want for the ARM team to review this PR, you must add the `ARMReview` label. 
- The automation may automatically add the `ARMReview` label, if appropriate.  
  If this happens, proceed according to guidance given in GitHub comments also added by the automation.

### Breaking change review

If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), 
follow the process outlined in the [High-level Breaking Change Process doc](https://aka.ms/breakingchangesprocess#high-level-breaking-change-process).
      
## Getting help

- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- For additional help, see https://aka.ms/azsdk/support/spectools.

### CODE REVIEW

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/examples/Events_ListBySingleResource.json
Comment (abhishek-corp) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/examples/Events_ListBySingleResource.json:96]: We don't need to show these new fields in this API example. The API flow is different for SingleResource events.

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (abhishek-corp) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1203]: Put better description so external users can understand the use case.
Something like
"This is a unique id for Planned Maintenance events used by some of our systems. "

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (abhishek-corp) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1205]: Put better description so external users can understand the use case.

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (abhishek-corp) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1211]: Put better description so external users can understand the use case.

**Review by abhishek-corp — CHANGES_REQUESTED** [review]

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (abhishek-corp) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1194]: Can we change this to something like this?

"Approx duration of impact in seconds. Mostly used for Planned Maintenance events."

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (abhishek-corp) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1199]: Can we change this to something like this?

"The type of impact. Mostly used for Planned Maintenance events."

Comment (mkatkam) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1199]: Keeping this as is for now, to avoid any  breaking changes.

**Review by abhishek-corp — APPROVED** [review]

**Comment by visingla-ms** [comment]
> Hi @mkatkam! The automation detected breaking changes in this pull request. As a result, it added the `BreakingChangeReviewRequired` label. You cannot proceed with merging this PR until you complete one of the following action items: **ACTION ITEM ALTERNATIVE A**: Fix the breaking change. Please consult the documentation provided in the relevant validation failures. **ACTION ITEM ALTERNATIVE B**: Request approval. Alternatively, if you cannot fix the breaking changes, then you can request an approval for them. Please follow the process described in the [High-level Breaking Change Process doc](https://aka.ms/breakingchangesprocess#high-level-breaking-change-process). **ACTION ITEM ALTERNATIVE C**: Report false positive. If you think there are no breaking changes, i.e. the validation should pass yet it fails, then proceed as explained in **ACTION ITEM ALTERNATIVE B**. This applies even if the breaking change tool fails with internal runtime error. In such case a manual breaking change review is necessary.

Please fix the breaking changes or please get a breaking changes approval from Azure Breaking Changes Reviewers <azbreakchangereview@microsoft.com> and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.

**Comment by mkatkam** [comment]
> > Hi @mkatkam! The automation detected breaking changes in this pull request. As a result, it added the `BreakingChangeReviewRequired` label. You cannot proceed with merging this PR until you complete one of the following action items: **ACTION ITEM ALTERNATIVE A**: Fix the breaking change. Please consult the documentation provided in the relevant validation failures. **ACTION ITEM ALTERNATIVE B**: Request approval. Alternatively, if you cannot fix the breaking changes, then you can request an approval for them. Please follow the process described in the [High-level Breaking Change Process doc](https://aka.ms/breakingchangesprocess#high-level-breaking-change-process). **ACTION ITEM ALTERNATIVE C**: Report false positive. If you think there are no breaking changes, i.e. the validation should pass yet it fails, then proceed as explained in **ACTION ITEM ALTERNATIVE B**. This applies even if the breaking change tool fails with internal runtime error. In such case a manual breaking change review is necessary.
> 
> Please fix the breaking changes or please get a breaking changes approval from Azure Breaking Changes Reviewers [azbreakchangereview@microsoft.com](mailto:azbreakchangereview@microsoft.com) and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.

**Comment by mkatkam** [comment]
> > Hi @mkatkam! The automation detected breaking changes in this pull request. As a result, it added the `BreakingChangeReviewRequired` label. You cannot proceed with merging this PR until you complete one of the following action items: **ACTION ITEM ALTERNATIVE A**: Fix the breaking change. Please consult the documentation provided in the relevant validation failures. **ACTION ITEM ALTERNATIVE B**: Request approval. Alternatively, if you cannot fix the breaking changes, then you can request an approval for them. Please follow the process described in the [High-level Breaking Change Process doc](https://aka.ms/breakingchangesprocess#high-level-breaking-change-process). **ACTION ITEM ALTERNATIVE C**: Report false positive. If you think there are no breaking changes, i.e. the validation should pass yet it fails, then proceed as explained in **ACTION ITEM ALTERNATIVE B**. This applies even if the breaking change tool fails with internal runtime error. In such case a manual breaking change review is necessary.
> 
> Please fix the breaking changes or please get a breaking changes approval from Azure Breaking Changes Reviewers [azbreakchangereview@microsoft.com](mailto:azbreakchangereview@microsoft.com) and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.

Hi, The breaking changes tag was added because I changed the description of a property, but currently I removed everything and kept as is. So, currently there are no breaking changes.

**Comment by TimLovellSmith** [comment]
>Hi, The breaking changes tag was added because I changed the description of a property,

But that doesn't really explain these errors does it?

>Rule	Message
❌ 1005 - RemovedPath	The new version is missing a path that was found in the old version. Was path '/{resourceUri}/providers/Microsoft.ResourceHealth/childAvailabilityStatuses/current' removed or restructured?
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L884:5
❌ 1005 - RemovedPath	The new version is missing a path that was found in the old version. Was path '/{resourceUri}/providers/Microsoft.ResourceHealth/childAvailabilityStatuses' removed or restructured?
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L931:5
❌ 1005 - RemovedPath	The new version is missing a path that was found in the old version. Was path '/{resourceUri}/providers/Microsoft.ResourceHealth/childResources' removed or restructured?
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L981:5
❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was 'articleId' renamed or removed?
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1563:11
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1710:11
❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was '_ActionUrl.Comment' renamed or removed?
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1748:7
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1899:7
❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was 'reason' renamed or removed?
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L987:11
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1134:11
❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was 'articleId' renamed or removed?
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1073:15
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1224:15
❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was 'parameters' renamed or removed?
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1073:15
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1224:15
⚠️ 1032 - DifferentAllOf	The new version has a different 'allOf' property than the previous one.
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1936:11
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L2091:11
⚠️ 1032 - DifferentAllOf	The new version has a different 'allOf' property than the previous one.
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L938:13
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1085:13
⚠️ 1032 - DifferentAllOf	The new version has a different 'allOf' property than the previous one.
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1946:5
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L2101:5
💬 1041 - AddedPropertyInResponse	The new version has a new property 'maintenanceId' in response that was not found in the old version.
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L987:11
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1134:11
💬 1041 - AddedPropertyInResponse	The new version has a new property 'maintenanceType' in response that was not found in the old version.
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L987:11
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1134:11
💬 1041 - AddedPropertyInResponse	The new version has a new property 'argQuery' in response that was not found in the old version.
New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L987:11
Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1134:11

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (TimLovellSmith) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1203]: >Id [](http://example.com/codeflow?start=37&length=2)

identifier?

Comment (TimLovellSmith) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1203]: Doc style - they should be "planned maintenance events" (all lowercase) - no need to uppercase this in English docs.

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (TimLovellSmith) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1211]: You should expand out the acronym ARG in case its not obvious to people in the context of Microsoft.ResourceHealth docs

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (TimLovellSmith) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1254]: I suggest a doc style/content change "It provides the name of portal extension to produce link for given service health event." to just "The name of the Azure portal extension to link to for the given service health event."

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (TimLovellSmith) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1258]: I suggest a doc style/content change to just "The name of the Azure portal extension blade to link to for the given service health event."

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (TimLovellSmith) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1260]: this object definition doesn't include a proper type declaration that explains how the mapping would be used by an API consumer.

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (TimLovellSmith) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1262]: typo 'lik' - I wonder if its not caught by spellcheck. Should it be link?

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/examples/Events_ListBySubscriptionId.json
Comment (TimLovellSmith) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/examples/Events_ListBySubscriptionId.json:102]: are these realistic enough values that you would want to show in your example docs?

**Comment by TimLovellSmith** [comment]
> > Hi, The breaking changes tag was added because I changed the description of a property,
> 
> But that doesn't really explain these errors does it?
> 
> > Rule	Message
> > ❌ 1005 - RemovedPath	The new version is missing a path that was found in the old version. Was path '/{resourceUri}/providers/Microsoft.ResourceHealth/childAvailabilityStatuses/current' removed or restructured?
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L884:5

From looking at the PR diff some more, looks like you based it off an old preview version 2022-10-01-preview instead of the latest stable version?

**Comment by mkatkam** [comment]
> > > Hi, The breaking changes tag was added because I changed the description of a property,
> > 
> > 
> > But that doesn't really explain these errors does it?
> > > Rule	Message
> > > ❌ 1005 - RemovedPath	The new version is missing a path that was found in the old version. Was path '/{resourceUri}/providers/Microsoft.ResourceHealth/childAvailabilityStatuses/current' removed or restructured?
> > > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L884:5
> 
> From looking at the PR diff some more, looks like you based it off an old preview version 2022-10-01-preview instead of the latest stable version? If that's just accidental probably you should fix it, so people can you all your features with the new preview.

Yes, I confirmed with my team as well that I should be using the latest stable version as the base version. So, I will create another PR with base version as 2022-10-01. Is there any other way I can use the same PR ?

**Comment by mkatkam** [comment]
> > Hi, The breaking changes tag was added because I changed the description of a property,
> 
> But that doesn't really explain these errors does it?
> 
> > Rule	Message
> > ❌ 1005 - RemovedPath	The new version is missing a path that was found in the old version. Was path '/{resourceUri}/providers/Microsoft.ResourceHealth/childAvailabilityStatuses/current' removed or restructured?
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L884:5
> > ❌ 1005 - RemovedPath	The new version is missing a path that was found in the old version. Was path '/{resourceUri}/providers/Microsoft.ResourceHealth/childAvailabilityStatuses' removed or restructured?
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L931:5
> > ❌ 1005 - RemovedPath	The new version is missing a path that was found in the old version. Was path '/{resourceUri}/providers/Microsoft.ResourceHealth/childResources' removed or restructured?
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L981:5
> > ❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was 'articleId' renamed or removed?
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1563:11
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1710:11
> > ❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was '_ActionUrl.Comment' renamed or removed?
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1748:7
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1899:7
> > ❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was 'reason' renamed or removed?
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L987:11
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1134:11
> > ❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was 'articleId' renamed or removed?
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1073:15
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1224:15
> > ❌ 1033 - RemovedProperty	The new version is missing a property found in the old version. Was 'parameters' renamed or removed?
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1073:15
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1224:15
> > ⚠️ 1032 - DifferentAllOf	The new version has a different 'allOf' property than the previous one.
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1936:11
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L2091:11
> > ⚠️ 1032 - DifferentAllOf	The new version has a different 'allOf' property than the previous one.
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L938:13
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1085:13
> > ⚠️ 1032 - DifferentAllOf	The new version has a different 'allOf' property than the previous one.
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L1946:5
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L2101:5
> > 💬 1041 - AddedPropertyInResponse	The new version has a new property 'maintenanceId' in response that was not found in the old version.
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L987:11
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1134:11
> > 💬 1041 - AddedPropertyInResponse	The new version has a new property 'maintenanceType' in response that was not found in the old version.
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L987:11
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1134:11
> > 💬 1041 - AddedPropertyInResponse	The new version has a new property 'argQuery' in response that was not found in the old version.
> > New: Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json#L987:11
> > Old: Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json#L1134:11

Yes, I think not using the latest stable version as the base version caused this. I will fix these by using the latest stable version.

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (mkatkam) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1203]: I will fix this. Thank you!

Comment (mkatkam) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1203]: I will fix this, thank you!

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (mkatkam) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1211]: I will expand ARG, thank you!

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (mkatkam) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1254]: Quick question on this - We want to correct this and make it better, but will it not create another breaking change as I am changing something that is already existing.?

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (mkatkam) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1258]: Since this is already existing, I am wondering if this will be a breaking change if I change this.

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/examples/Events_ListBySubscriptionId.json
Comment (mkatkam) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/examples/Events_ListBySubscriptionId.json:102]: These are not realistic values, will try to add sample values for the examples. Thank you for your suggestion.

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json
Comment (mkatkam) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json:1262]: I will fix this,  but will check if this will cause any breaking changes if I change this. And then proceed accordingly.

**Comment by mkatkam** [comment]
Closing this PR as this is incorrectly based off of 2022-10-01-preview instead of 2022-10-01.
Original and new PR https://github.com/Azure/azure-rest-api-specs/pull/25447

**Pull request closed without merging**
